### PR TITLE
add pelican content in blog/

### LINKED ISSUE #159: Create a website/blog
Referring what Marc suggested in his last email,
`I think it could be nice if you want to create a website, a blog... to show your work, share what you're learning, and helping other people who can be interested in also contributing to pandas or other open source projects`

I have set up a GitHub Page to host the `pandas-mentoring` repo 
https://python-sprints.github.io/pandas-mentoring/ 

Other suggestions on creating a website/blog are also welcome.

### PR DESCRIPTION
**Pull request by dujm**
Close #159 

I add a `blog/` to host the content of pandas-mentoring website.

The website is generated using Pelican.

I plan to publish the html sites o `gh-pages branch`

### CODE REVIEW

**Review by datapythonista — APPROVED** [review]
lgtm

Did you leave the post about creating a blog in pelican intentionally? May be useful, but not sure if it's just part of the pelican template and that's why it's in the PR.

@python-sprints/pandas-mentoring can someome please review so @dujm can move forward with this?

File: blog/content/build-website.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+## 4. Generate and preview your site
+```
+pelican content
+pelican --listen
+# Navigate to http://localhost:8000/ in your browser.
+```
+
+## 5. Add a theme
+```
+# Choose a path for to clone pelican-themes directory (e.g. blog/)
```
Comment (sara-02) [inline blog/content/build-website.md]: `for to` maybe a typo?

File: blog/content/contributing.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+   # 1) Go to the "master" branch of your fork ("origin")
+    git checkout master
+
+   # 2) Fetch the commits from the "upstream"
+    git fetch upstream
+
+   # 3) If you have local changes, stash the changes of your "master" branch
+    git stash
+
+   # 4) Merge the changes from the "master" branch of the "upstream" into your the "master" branch of your "origin"
```
Comment (sara-02) [inline blog/content/contributing.md:47]: `your the` minor typo here as well.

File: blog/pelicanconf.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+# Add a theme
+THEME = '../pelican-themes/uikit'
+DISPLAY_TAGS_ON_SIDEBAR_LIMIT = 0
+DISPLAY_LINKS_ON_SIDEBAR_LIMIT = 0
+LICENSE = {
+    'cc_name':"by-sa",
+    'hosted':False,
+    'compact':True,
+    'brief':False
+    }
```
Comment (sara-02) [inline blog/pelicanconf.py]: Adding a space after `:` would be closer to python formatting. Eg: `'cc_name':  "by-sa"`

**Review by sara-02 — COMMENTED** [review]
Minor comments.
If you wish to bypass these instead, let me know and I will approve.

**Comment by dujm** [comment]
> Minor comments.
> If you wish to bypass these instead, let me know and I will approve.
Hi @sara-02, thanks for reviewing! I will update the files and resend the pull request.

**Comment by dujm** [comment]
> lgtm
> 
> Did you leave the post about creating a blog in pelican intentionally? May be useful, but not sure if it's just part of the pelican template and that's why it's in the PR.
> 
> @python-sprints/pandas-mentoring can someome please review so @dujm can move forward with this?

Hi @datapythonista, yes, I left the `build-website.md` intentionally. I will later put it in an appropriate group with other blog posts.

**Comment by dujm** [comment]
Hi @sara-02, could you have another review at the files before I merge them? Thanks!

**Comment by dorothykiz1** [comment]
lgtm

**Review by sara-02 — APPROVED** [review]
LGTM

**Pull request merged**